google-cloud-tools-plugin/src/com/google/gct/idea/appengine/validation/FullJavaNameInspection.java

```diff
@@ -97,15 +97,23 @@ private void validateBackendMethodNameUnique(PsiMethod psiMethod, Map<String, Ps
           return;
         }
 
-        String javaName = psiMethod.getContainingClass().getQualifiedName() + "." + psiMethod.getName();
+        PsiClass containingClass = psiMethod.getContainingClass();
```

Comment:
Actually I am not sure about this one and was hoping to get feedback. If it seems logically impossible, then I'll leave it as-is.